src/Benchmarks/FortunesEfMiddleware.cs

```diff
@@ -44,9 +44,7 @@ public FortunesEfMiddleware(RequestDelegate next)
 
         private static async Task<IEnumerable<Fortune>> LoadRows(ApplicationDbContext dbContext)
         {
-            var result = new List<Fortune>();
-
-            result.AddRange(await dbContext.Fortunue.ToListAsync());
+            var result = await dbContext.Fortune.ToListAsync();
```

Comment:
EF doesn't do anything special here. The ToList/Async operators are actually outside of EF - In Linq-to-objects and IX-Async respectively.